java-checks/src/main/java/org/sonar/java/checks/helpers/UnitTestUtils.java

```diff
@@ -34,6 +36,38 @@
 
 public final class UnitTestUtils {
 
+  public static final Pattern ASSERTION_METHODS_PATTERN = Pattern.compile("(assert|verify|fail|should|check|expect|validate).*");
+  public static final Pattern TEST_METHODS_PATTERN = Pattern.compile("test.*|.*Test");
+
+  public static final MethodMatchers ASSERTION_INVOCATION_MATCHERS = MethodMatchers.or(
+    // fest 1.x / 2.X
+    MethodMatchers.create().ofSubTypes("org.fest.assertions.GenericAssert", "org.fest.assertions.api.AbstractAssert").anyName().withAnyParameters().build(),
+    // rest assured 2.0
+    MethodMatchers.create().ofTypes("io.restassured.response.ValidatableResponseOptions")
+      .name(name -> name.equals("body") ||
+        name.equals("time") ||
+        name.startsWith("time") ||
```

Comment:
This case is not covered in tests because as I found there is only one method starting with "time", which is time itself and it is true for `name.equals("time")` if I remove this line, could hit performance a little(very little), I wanted to remove `startsWith(..)` but maybe I am missing something

Reply:
It might be related to old version of restassured API, and so it would make us support them all, to be confirmed. For me it's OK to let it like this.